Topic1_Arrays/Day2207/Day2207_Tuan/1_TwoSum.txt

```diff
@@ -0,0 +1,16 @@
+//Complexity: O(n)
+
+class Solution {
+public:
+    vector<int> twoSum(vector<int>& nums, int target) {
+        unordered_map<int, int> mp;

```

Comment:
Add the way you think to solve the problem, it will help you improve skill to explain your idea. It helps when you take the interview.
Please add more meaningful variable name